dashboard/job-pulse-kanban.html

```
@@ -0,0 +1,418 @@
+    <button onclick="exportJson()">⬆ Export JSON</button>
+    <label style="cursor:pointer"><button as="span" onclick="document.getElementById('import-file').click()">⬇ Import JSON</button>
+      <input id="import-file" type="file" accept=".json" style="display:none" onchange="importJson(event)"></label>
+    <button onclick="clearAll()" style="color:var(--red)">✕ Clear</button>
+  </div>
+</header>
+
+<div class="board" id="board"></div>
+
+<div class="modal-bg" id="detail-modal">
+  <div class="modal">
+    <button class="modal-close" onclick="closeModal()">✕</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)